Model.elm

```diff
@@ -290,18 +359,18 @@ update msg ({ shows, formData } as model) =
                     | shows = updatedShows
                     , allGenres = extractAllGenres updatedShows
                   }
-                , saveShows updatedShows
+                , saveBackup authToken updatedShows
```

Comment:
Not really because it gets really ugly with large statements once indented by elm-format.